src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -354,7 +374,7 @@ protected function doRequestInProcess($request)
             unlink($deprecationsFile);
             foreach ($deprecations ? unserialize($deprecations) : array() as $deprecation) {
                 if ($deprecation[0]) {
-                    trigger_error($deprecation[1], E_USER_DEPRECATED);
+                    @trigger_error($deprecation[1], E_USER_DEPRECATED);
```

Comment:
this change is adding a bug, fabbot is not always right

Reply:
will revert